Podfile

```diff
@@ -0,0 +1,17 @@
+# Uncomment the next line to define a global platform for your project
+ platform :ios, '9.0'
+
+target 'companion' do
+  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for companion
+
+    pod 'iBeaconManager'
```

Comment:
Since we're not committing the Pod folder, please add a specific version to all Pods